app/models/organisation.rb

```diff
@@ -81,9 +81,21 @@ def self.search_by_keyword(keyword)
      self.where(contains_description(keyword).or(contains_name(keyword)))
   end
 
-  def self.filter_by_category(category_id)
-    return all unless category_id.present?
-    self.joins(:categories).where(is_in_category(category_id)) #do we need to sanitize category_id?
+  def self.filter_by_categories(category_ids)
```

Comment:
not sure the diff highlights precisely that change ... or I'm going code blind :-)